vendor/zkllvm-metacraft-circuits/scripts/compile_and_run_tests.sh

```diff
@@ -14,6 +14,11 @@ then
     git clone git@github.com:metacraft-labs/finalizer-data.git $SCRIPT_DIR/../src/tests/verify_attestation_data_test/finalizer-data
 fi
 
+if [ ! -d $SCRIPT_DIR/../../consensus-spec-tests ]
+then 
+    git clone git@github.com:ethereum/consensus-spec-tests.git $SCRIPT_DIR/../../consensus-spec-tests
+fi
```

Comment:
Use environment variable for the tests directory or make it relative to the project root so that you don't have to repeat yourself